veg2hab/habitat.py

```diff
@@ -13,21 +14,25 @@
 from veg2hab.vegetatietypen import VvN as _VvN
 
 
-@dataclass
-class HabitatVoorstel:
+class HabitatVoorstel(BaseModel):
     """
     Een voorstel voor een habitattype voor een vegetatietype
     """
 
-    onderbouwend_vegtype: Optional[Union[_SBB, _VvN]]
-    vegtype_in_dt: Optional[Union[_SBB, _VvN]]
+    class Config:
+        extra = "forbid"
+
+    onderbouwend_vegtype: Union[_SBB, _VvN, None]
+    vegtype_in_dt: Union[_SBB, _VvN, None]
     habtype: str
     kwaliteit: Kwaliteit
     idx_in_dt: Optional[int]
     mits: BeperkendCriterium
     mozaiek: MozaiekRegel
     match_level: MatchLevel
-    mozaiek_dict: Optional[dict] = None
+    # TODO: I turned this into a list of tuples instead of a dict
+    # maybe turn it into a named tuple
+    mozaiek_dict: Optional[List[Tuple[str, bool, Kwaliteit, float]]] = None
```

Comment:
@JRdeLange, hier moet jij ff wat van vinden. Het is nu niet netjes opgelost.

Reply:
Ik zeg of dikke prima zo, of misshcien al de omzetting naar string naar voren halen, dat hij hier nooit als dict aankomt.

EDIT zie comment bij vegkartering.py line 270